src/static/css/bootstrap-table-accordion.css

```diff
@@ -1,123 +1,310 @@
-* https://github.com/wenzhixin/bootstrap-table
-*
-************************************/
-
-/* restore thicker bootstrap header bottom border removed in bootstrap-table */
+/*=============================================================
+=            Bootstrap Table Accordion Styling Fixes          =
+=         https://github.com/wenzhixin/bootstrap-table        =
+
+=  In Bootstrap 3, many important styling rules for the
+=  tables are missing, such as danger, warning, info etc.
+
+=  In addition, this file contains many styles for the quest
+=  list accordion table, making it easier to read and use.
+==============================================================*/
+
+/* https://github.com/wenzhixin/bootstrap-table */
+
+/* Adjusting bootstrap header border for bootstrap-table */
 .table > thead > tr > th {
     border-bottom-width: 2px !important;
-}
+  }
 
-/* remove the transparent border to prevent gaps in row borders */
-.bootstrap-table .table.table-no-bordered > tbody > tr > td,
-.bootstrap-table .table.table-no-bordered > thead > tr > th {
+  /* Remove unnecessary transparent border to prevent gaps in row borders */
+  .bootstrap-table .table.table-no-bordered > tbody > tr > td,
+  .bootstrap-table .table.table-no-bordered > thead > tr > th {
     border-right: none;
     border-left: none;
-}
+  }
 
-/* New contextual class: .primary */
-.table>tbody>tr.primary>td, .table>tbody>tr.primary>th, .table>tbody>tr>td.primary, .table>tbody>tr>th.primary,
-.table>tfoot>tr.primary>td, .table>tfoot>tr.primary>th, .table>tfoot>tr>td.primary, .table>tfoot>tr>th.primary,
-.table>thead>tr.primary>td, .table>thead>tr.primary>th, .table>thead>tr>td.primary, .table>thead>tr>th.primary {
-    background-color: #337ab7 !important;
-    color: #fff;
-    border-color: #337ab7;
-}
+  /* Adjusting padding for sorting icons in header */
+  .fixed-table-container thead th .sortable {
+    padding-right: 18px;
+  }
 
-.table>tbody>tr.primary a {
-  color: #173652;
-}
 
+  /*=============================================
+  =            Accordion Table Styles           =
+  =============================================*/
 
-/*.table>tbody>tr.row-warning, .table>tbody>tr.row-info:hover{*/
-    /*color: #8a6d3b;*/
-    /*background-color: #fcf8e3;*/
-    /*border-color: #faebcc;*/
-/*}*/
+  /* Hiding the first column to clean up the table presentation */
+  .accordian-table thead>tr>th.detail,
+  .accordian-table tbody>tr:not(.detail-view)>td:first-of-type {
+    display: none;
+  }
+
+  /* Removing border from inner cells for a cleaner appearance */
+  .accordian-table.table-bordered > tbody > tr > td,
+  .accordian-table.table-bordered > thead > tr > th {
+    border: none;
+  }
+
+  /* Removing border from the table header */
+  .fixed-table-container,
+  .bootstrap-table .table > thead > tr > th,
+  .table > thead > tr > th,
+  .table > tbody > tr > th {
+    border: none;
+  }
+
+  /* Removing spacing between expanded accordion rows for a cleaner flow */
```

Comment:
I don't think this extra CSS is necessary.  The gap is caused by the buttons have a bottom margin of 15px (caused by `.list-group-item-buttons > .pull-right` in custom_common.css), but the containing element only having a bottom padding of 10px (caused by .list-group-item class and provided by bootstrap).

So if the content is only a single line of text, the margin will push 5px past the padding, causing the gap.  If there is more than one line of text, then the container will be large enough to contain the button margin

I think a more simple fix is to just add 5px to the minimum-height of the containing element, OR reduce the margin of the buttons by 5px.  The margin is important to give a gap below the buttons in case there is lots of text that will flow below the button. Both of these styles are in our custom_common.css.

I think the better solution is to set margin-bottom: 10 px; on the buttons, via classes shown here:

![image](https://github.com/bytedeck/bytedeck/assets/10604391/fdf5278f-be00-496a-b0ae-46ee8c18cffd)

Reply:
You were right! I tried changing the min-height on a few other elements, but I never got it to work. But changing it on that `.list-group-item-buttons` class seemed to do the trick.

```css
/* custom_common.css */

/*
The .list-group-item-buttons rule sets a minimum height of 65px.
This is necessary to eliminate a small gap that appears between an expanded
bootstrap-table accordion row and the subsequent row. This gap is caused by a
15px margin-bottom on the .pull-right child element of .list-group-item-buttons.
*/
.list-group-item-buttons {
    min-height: 65px;  /* used to be 55px */
}

.list-group-item-buttons > .pull-right {
    margin-left: 15px;
    margin-bottom: 15px;
}
```

This allowed me to remove the classes you mentioned.